app/models/viewmodels/notificationspage/NotificationPageSingleElementFactory.scala

```diff
@@ -34,14 +34,16 @@ class NotificationPageSingleElementFactory @Inject()(responseConverterProvider:
 
   def build(submission: Submission)(implicit messages: Messages): NotificationsPageSingleElement =
     submission.actionType match {
-      case Arrival | RetrospectiveArrival | Departure | DucrDisassociation | MucrAssociation | MucrDisassociation | ShutMucr =>
+      case Arrival | RetrospectiveArrival | Departure | DucrDisassociation | DucrPartDisassociation | MucrAssociation | MucrDisassociation |
+          ShutMucr =>
         buildForRequest(submission)
-      case DucrAssociation => buildForDucrAssociation(submission)
+      case DucrAssociation     => buildForDucrAssociation(submission)
```

Comment:
Why is this different to ext movements?  There both DucrAssociation and DucrPartAssociation -> buildForDucrAssociation()

Reply:
This is the bug in the External Movements service, I'll create a fix for it there after I merge internal

Reply:
OK.  Great!